test/IECoreMaya/LiveSceneTest.py

```diff
@@ -990,6 +990,21 @@ def testSceneShapeVisible( self ) :
 		maya.cmds.setAttr( envShape + ".visibility", False )
 		self.assertEqual( envScene.readAttribute( "scene:visible", 0 ), IECore.BoolData( False ) )
 	
+	def testMultiShapes( self ) :
```

Comment:
Since we're supporting mult-shapes for meshes too, can we test merging them as well? And make sure intermediates don't get merged?